js/helpers.js

```diff
@@ -24,452 +24,448 @@
 //  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
 //  OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 //  OF THE POSSIBILITY OF SUCH DAMAGE.
-define(["./globals", 'underscore', "jquery", "jquerySizes", "./models/spine_item", 'URIjs'], function(Globals, _, $, JQuerySizes, SpineItem, URI) {
```

Comment:
It looks like most (all?) of the code changes below relate to automatic code formatting (VSCode I presume?). To avoid confusing the objective of this Pull Request, such changes should be reverted (even if I personally love consistent formatting and wish there was a super-strict lint checker ;)